src/hook/useCustomRefForm.js

```diff
@@ -0,0 +1,44 @@
+import { useRef, useState } from "react";
+
+const useCustomRefForm = (initialInput, validate) => {
+  const inputRefs = useRef(initialInput);
```

Comment:
why we are using useRef ?

Reply:
useRef is used here to get direct access to the element from the dom.
It reduces the rendering of the element while updation